docs/community/index.md

```diff
@@ -7,7 +7,7 @@ examples, you are welcome to help develop and improve napari.
 
 To ask questions and interact with the team, you can join our
 [zulip chat](https://napari.zulipchat.com/login/) or participate in our
-[meetings](./meeting_schedule).
+[meetings](meeting-schedule).
 
 If you are interested in contributing, check out our
-[contributing guide](../developers/index).
+[contributing guide](contributing).
```

Comment:
I don't get why the link works in the artifact, I'd think that this tag needs to be added to https://github.com/napari/docs/tree/main/docs/developers/index.md